src/main/java/org/launchcode/workforce/repository/StaffRepository.java

```diff
@@ -0,0 +1,12 @@
+package org.launchcode.workforce.repository;
+
+import org.launchcode.workforce.model.ClientsJobs;
```

Comment:
You don't need this ClientsJobs import statement,

Reply:
done - thank you